CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # [develop](https://github.com/adhearsion/electric_slide)
 
+* Bugfix: `FixedPriority` `#checkout_agent` now returns the agent with highest priority
```

Comment:
I would try to make this more friendly, like "Bugfix: FixedPriority now correctly checks out agents in priority order."